theme/less/_navigation.less

```diff
@@ -1,26 +1,70 @@
-nav
+header nav
 {
-  ul
+  border-top: 1px solid #E9E9E9;
+  background: #FFFFFF;
+  box-shadow: 0px 8px 15px rgba(0, 0, 0, 0.06);
+}
```

Comment:
I think you won't need this style after moving relevant header style out of the mobile version, as mentioned above.